simulation.py

```diff
@@ -0,0 +1,150 @@
+import numpy as np
+import pandas as pd
+import random
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+# Number of excitatory and inhibitory neurons
+N_E = 80
+N_I = 20
+n_neurons = N_E + N_I
+n_sessions = 6
+total_time = 3000
+
+# All the parameters from Supplementary table from the paper.
+W_EI = 0.44
+W_IE = 0.66
+W_II = 0.54
+W_EE = 0.37
+W_EI2 = 0.49
+W_IE2 = 0.65
+W_II2 = 0.53
+
+mu_EI = W_EI
+mu_IE = W_IE
+mu_II = W_II
+sigma_EI2 = W_EI2 - W_EI ** 2
+sigma_IE2 = W_IE2 - W_IE2 ** 2
+sigma_II2 = W_II2 - W_II ** 2
+sigma_EI = np.sqrt(sigma_EI2)
+sigma_IE = np.sqrt(sigma_IE2)
+sigma_II = np.sqrt(sigma_II2)
+print(str(sigma_EI2) + '\n' + str(sigma_IE2) + '\n' + str(sigma_II2))
+
+# Extracting E->E connectivity from the spine imaging data
+c_EE = 0.2
+path = "Global_Spines_info.csv"
+spines_info = pd.read_csv(path)
+spines_info.drop('Unnamed: 0', axis=1, inplace=True)
+spines_info.head()
+
+spines_IS1 = spines_info.loc[spines_info['Starting Imaging Session'] == 1]
+spines_IS1.head(100)  # Why 100 here?
```

Comment:
100 is just for myself to check whether I got the right data from imaging session one. It could be changed to any value.